.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
nit: I think we don't need to add .gitignore to the fork, we can try to minimize changes to upstream.

Reply:
done